packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -169,8 +169,22 @@ export class ListItem extends LitElement implements InteractiveComponent, Sortab
    */
   @property() moveToItems: MoveTo[] = [];
 
-  /** When `true`, the item is open to show child components. */
-  @property({ reflect: true }) open = false;
+  /**
+   * When `true`, the item is open to show child components.
+   *
+   * @deprecated Use `expanded` prop instead.
```

Comment:
Can you update external usage of `open` too? `list` has a [few](https://github.com/Esri/calcite-design-system/blob/904623be3cff426e71d51001a29df188a72ea00b/packages/calcite-components/src/components/list/list.tsx#L884) [usages](https://github.com/Esri/calcite-design-system/blob/904623be3cff426e71d51001a29df188a72ea00b/packages/calcite-components/src/components/list/list.tsx#L631). @driskull Are there any other spots that need updating?

Reply:
They should all just be in that file I would suspect.